[Bug]: mdbook-cat-prep build failed

### Problem description

```
error[E0[793](https://github.com/johndebia2022/termux-packages/actions/runs/5064173739/jobs/9091511992#step:6:794)]: reference to packed field is unaligned
   --> /home/builder/.cargo/registry/src/github.com-1ecc6299db9ec823/tendril-0.4.1/src/tendril.rs:241:20
    |
241 |                 if (*header).refcount.decrement() == 1 {
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: fields of packed structs are not properly aligned, and creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)
    = help: copy the field contents to a local variable, or replace the reference with a raw pointer and use `read_unaligned`/`write_unaligned` (loads and stores via `*p` must be properly aligned even when using raw pointers)

error[E0793]: reference to packed field is unaligned
   --> /home/builder/.cargo/registry/src/github.com-1ecc6299db9ec823/tendril-0.4.1/src/tendril.rs:958:9
    |
958 |         (*self.header()).refcount.increment();
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: fields of packed structs are not properly aligned, and creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)
    = help: copy the field contents to a local variable, or replace the reference with a raw pointer and use `read_unaligned`/`write_unaligned` (loads and stores via `*p` must be properly aligned even when using raw pointers)

For more information about this error, try `rustc --explain E0793`.
error: could not compile `tendril` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
Error: Process completed with exit code 101.
```



[2.txt](https://github.com/termux/termux-packages/files/11549999/2.txt)


### What steps will reproduce the bug?

build

### What is the expected behavior?

_No response_

### System information

termux-info:

```

```

--- a/packages/mdbook-cat-prep/build.sh
+++ b/packages/mdbook-cat-prep/build.sh
@@ -9,6 +9,9 @@ TERMUX_PKG_SRCURL=git+https://github.com/gjk-cat/cat-prep
 TERMUX_PKG_GIT_BRANCH=master
 TERMUX_PKG_BUILD_IN_SRC=true
 
+# https://github.com/termux/termux-packages/issues/16756
+TERMUX_RUST_VERSION=1.68.2
+
 termux_step_post_get_source() {
 	git fetch --unshallow
 	git checkout $_COMMIT

